src/display/annotation_layer.js

```diff
@@ -760,7 +767,9 @@ var AnnotationLayer = (function AnnotationLayerClosure() {
           linkService: parameters.linkService
         };
         var element = annotationElementFactory.create(properties);
-        parameters.div.appendChild(element.render());
+        if (element && element.isRenderable) {
```

Comment:
We do not need to check for `isRenderable` here anymore if we return early in the element's constructor.